src/sdk/namespace_fs.js

```diff
@@ -79,6 +79,45 @@ function sort_entries_by_name(a, b) {
     return 0;
 }
 
+function _get_mtime_from_filename(filename) {
+    if (!filename.includes('_mtime-')) {
+        // Latest file wont have time suffix which will push the latest
+        // object last in the list. So to keep the order maintained,
+        // returning the latest time. Multiplying with 1e6 to provide
+        // nano second precision
+        return BigInt(Date.now() * 1e6);
+    }
+    const file_parts = filename.split('-');
+    return size_utils.string_to_bigint(file_parts[file_parts.length - 3], 36);
+}
+
+function _get_filename(file_name) {
+    if (file_name.includes('_mtime-')) {
+        return file_name.substring(0, file_name.indexOf('_mtime-'));
+    }
+    return file_name;
+}
+/**
+ * @param {fs.Dirent} a
+ * @param {fs.Dirent} b
+ * @returns {Number}
+ */
+function sort_entries_by_name_and_time(a, b) {
```

Comment:
please use meaningful variable names.

Reply:
variables... 
don't use `a` and `b`.
and corresponding in the function itself, `a_name` 
rather be something like `first_entry` `second_entry` or something similar

Reply:
sure